src/js/app.js

```
@@ -1,24 +1,104 @@
+    updateAchievementsTable(clicks);
+}
 
 // Detect if the website is loaded on a desktop
 const isDesktop = window.innerWidth > 1024;
 
-
 const clickCounterElement = document.getElementById("click-counter");
 const fullscreenButton = document.getElementById("fullscreen-button");
 
-let clickCounter = parseInt(localStorage.getItem('clickCount') || '0');
+let clickCounter = parseInt(localStorage.getItem("clickCount") || "0");
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)